datadog/resource_datadog_dashboard.go

```diff
@@ -4491,7 +4491,10 @@ func buildDatadogTimeseriesRequests(terraformRequests *[]interface{}) *[]datadog
 		if terraformMetadataList, ok := terraformRequest["metadata"].([]interface{}); ok && len(terraformMetadataList) > 0 {
 			datadogMetadataList := make([]datadogV1.TimeseriesWidgetRequestMetadata, len(terraformMetadataList))
 			for i, m := range terraformMetadataList {
-				metadata := m.(map[string]interface{})
+				metadata, ok := m.(map[string]interface{})
+				if !ok {
+					continue
+				}
```

Comment:
any chance this can create non empty plans ? If something is specified in the config but we completely ignore it ?

Reply:
Let's add an empty value in one of the test configs to try it out too